guide/src/README.md

```diff
@@ -1,13 +1,22 @@
 # Introduction
 
-> cargo, make me a project
+```rust
+cargo generate /me/a-project
+```
 
 [![Build status](https://github.com/cargo-generate/cargo-generate/workflows/Build/badge.svg)](https://github.com/cargo-generate/cargo-generate/actions?query=workflow%3ABuild+branch%3Amain+)
 [![crates.io](https://img.shields.io/crates/v/cargo-generate.svg)](https://crates.io/crates/cargo-generate)
 [![dependency status](https://deps.rs/repo/github/cargo-generate/cargo-generate/status.svg)](https://deps.rs/repo/github/cargo-generate/cargo-generate)
 
-`cargo-generate` is a developer tool to help you get up and running quickly with a new Rust
-project by leveraging a pre-existing git repository as a template.
+`cargo-generate` is a Rust developer tool to quickly setup any Rust project,
+by automating repetitive or intricate tasks.
+
+Project templates are ***any*** files.  Alternatively, files can leverage
+[Shopify's Liquid](http://liquidmarkup.org/) templates,
+[Rhai](https://docs.rs/rhai/latest/rhai/) scripts and Rust's 
+[regex crate](https://docs.rs/regex/latest/regex/) to add crates,
+features, initializers, etc. to your freshly created Rust project or an 
```

Comment:
> to your freshly created Rust project or an existing Rust project.

That again would make it very rust specific, but the `cargo-generate` is not really limited to that.